tests/test_github_start_date.py

```diff
@@ -103,19 +126,11 @@ def run_test(self, date_1, date_2, streams):
         record_count_by_stream_2 = self.run_and_verify_sync(conn_id_2)
         synced_records_2 = runner.get_records_from_target_output()
 
-        # Verify the total number of records replicated in sync 1 is greater than the number
-        # of records replicated in sync 2
+        # verify that sync 2 has at least one record synced and less records than sync 1
+        self.assertGreater(sum(record_count_by_stream_2.values()), 0)
```

Comment:
This assertion should be applied on individual stream as well.

Reply:
Added "verify that sync 2 has at least one record synced" assertion for individual stream.